app/src/main/java/com/hover/stax/balances/BalanceAdapter.java

```diff
@@ -43,32 +43,34 @@ public void showBalance(boolean show) {
 	@NonNull
 	@Override
 	public BalanceViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
-		View view = LayoutInflater.from(parent.getContext()).inflate(R.layout.balance_item, parent, false);
-		return new BalanceViewHolder(view);
+		BalanceItemBinding binding = BalanceItemBinding.inflate(LayoutInflater.from(parent.getContext()), parent, false);
+		return new BalanceViewHolder(binding);
 	}
 
 	@RequiresApi(api = Build.VERSION_CODES.LOLLIPOP)
 	@Override
 	public void onBindViewHolder(@NonNull BalanceViewHolder holder, int position) {
 		Channel channel = channels.get(position);
 
-		holder.channelName.setText(channel.name);
-		holder.channelId.setText(Integer.toString(channel.id));
+		holder.binding.balanceChannel.setText(channel.name);
```

Comment:
What of: ...balanceChannelName being a better naming?

Reply:
Agreed. Change effected.